junction/proposals/views.py

```diff
@@ -133,9 +134,27 @@ def create_proposal(request, conference_slug):
 
 
 @require_http_methods(['GET'])
-def detail_proposal(request, conference_slug, slug):
-    conference = get_object_or_404(Conference, slug=conference_slug)
-    proposal = get_object_or_404(Proposal, slug=slug, conference=conference)
+def detail_proposal(request, conference_slug, slug, hashid=None):
+    """Display a proposal detail page.
+    """
+    # Here try to get a proposal by it's hashid. If the slug didn't match becase
+    # the title might have change redirect to the correct slug.
```

Comment:
- Please make spelling correction for `becase`.
- Change `change redirect` to `changed, redirect`.